### DISCUSSION TITLE
If I have multiple roles linked to multiple levels, once someone reaches a higher level, is there a way for the old role to be removed?

### DISCUSSION THREAD

**Discussion by FiberzYT** [danthe1st/Mee6-bypasser#37] [Ideas]
**Describe the question**
Is there a way to remove an old role automatically if someone reaches a level in which they get a new one?

**Comment by danthe1st** [comment]
This is (currently) not implemented but I may implement it at some point.

**Comment by FiberzYT** [comment]
> This is (currently) not implemented but I may implement it at some point.

Ok, thanks for the reply, let me know if this will be implemented because it will be an amazing feature!

**Comment by artscr7** [comment]
Even I had this problem, please can you implement asap?

**Comment by danthe1st** [comment]
I currently don't have much time due to a project that is part of my school leaving exam (I need to do that in my free time) but I will try to implement it if I have time.
If someone wants to speed up that feature, I am accepting pull requests.

**Comment by danthe1st** [comment]
I modified the code so that old roles are removed but it might take a while until the bot is updated.
No additional configuration is required, old roles will just be removed.

**Comment by FiberzYT** [comment]
> I modified the code so that old roles are removed but it might take a while until the bot is updated.
> No additional configuration is required, old roles will just be removed.

Will I have to remove and re-add the bot for it to update, or will it do it automatically?

**Comment by danthe1st** [comment]
It will do automatically, as soon as it is updated.
